src/lib.rs

```diff
@@ -181,17 +181,28 @@ pub trait PrimeField: Field + From<u64> {
     const NUM_BITS: u32;
 
     /// How many bits of information can be reliably stored in the field element.
+    ///
+    /// This is usually `Self::NUM_BITS - 1`.
     const CAPACITY: u32;
 
-    /// Returns the multiplicative generator of `char()` - 1 order. This element
-    /// must also be quadratic nonresidue.
+    /// Returns the multiplicative generator of `modulus - 1` order. This element must
```

Comment:
`the generator` -> `a fixed generator` as most nonzero field elements are generators of the entire multiplicative subgroup. Sagemath just finds the first one it can when you call `.primitive_element()`.

Note, the choice of this generator impacts the choice of `root_of_unity` which is derived from it. If `root_of_unity` happens to be a different 2^s root of unity, this will break consistency with e.g. the Sapling/Sprout zk-SNARK parameters, which encode evaluations of Lagrange basis polynomials over the domain derived from `root_of_unity`.